test/metabase/pulse/render/js_svg_test.clj

```diff
@@ -47,7 +47,7 @@
     (is (.hasAttribute line "fill-opacity"))
     (is (= (.getAttribute line "fill-opacity") "0.0"))))
 
-(def ^Context context (delay (#'js-svg/static-viz-context)))
+(def ^Context context (@#'js-svg/context))
```

Comment:
```suggestion
(def ^Context context (#'js-svg/context))
```
I think the idiomatic way is just calling the var. In this form one might think that delays are involved.